resources/themes/cura-dark/theme.json

```diff
@@ -87,8 +94,8 @@
         "action_button_active": [39, 44, 48, 30],
         "action_button_active_text": [255, 255, 255, 255],
         "action_button_active_border": [255, 255, 255, 100],
-        "action_button_disabled": [19, 24, 28, 255],
-        "action_button_disabled_text": [200, 200, 200, 80],
+        "action_button_disabled": [85, 85, 87, 255],
+        "action_button_disabled_text": [103, 103, 104, 255],
```

Comment:
Are you sure the intent wasn't to just reduce the opacity for this one?

Reply:
It was not that way in the light theme. To have the light theme have an opaque disabled text and then have the dark theme have it be semi-translucent is just asking for trouble.